### ISSUE TITLE
User searching always takes you to page 208, while not filtering results

### ISSUE THREAD

**Issue by owainjones74** [everyday-as/gmodstore-issues#1108]
### Expected Behavior

When searching a key phrase, it should filter out all irrelevant results.

### Actual Behavior

It seems to not filter the results at all, and instead just take you to page 208.

### Reproduction Steps

```markdown
1. Go to https://www.gmodstore.com/users/browse
2. Search for anything. In my case, I search for "Owain".
3. The outcome is you're taken to https://www.gmodstore.com/users/browse?page=208
```

### Browsers

Chrome, but I assume it's the same on all of them?

### Example URL

https://www.gmodstore.com/users/browse

### GmodStore Version

v10.0.24

### Additional Information

Regardless of changing the "Sort by" or "Registered" filters, the result is always the same, page 208.

**Comment by Marcuzz** [comment]
Thanks for your report, I can confirm that this issue happens on Firefox as well. It's definitely a server-side issue.

Just some technical observations:

The strange thing is that while it clearly happens in production, it does not happen in a local setup. This indicates to me that it's an issue with either Traefik or Octane.

**Comment by Marcuzz** [comment]
Josh pushed a fix in 10.0.26 - this should work as expected now 🙂
